kyverno-policy-deployment/README.md

```diff
@@ -1,79 +1,7 @@
-# Handling Kyverno Policy Deployment
```

Comment:
This page is modified to be an index to the two kyverno policy management related RFCs. This content has been moved to policy-deployment.md. See sharing-policies.md for substantive content.